ambry-router/src/main/java/com/github/ambry/router/NonBlockingRouter.java

```diff
@@ -512,7 +512,10 @@ void initiateChunkDeletesIfAny(final String blobIdStr, final String serviceId,
         }
         initiateBackgroundDeletes(deleteRequests);
         if (blobMetadataCache != null) {
-          blobMetadataCache.deleteObject(blobIdStr);
+          boolean deleteResult = blobMetadataCache.deleteObject(blobIdStr);
+          logger.debug("[{}] Issued delete-metadata for blobId = {}, reason = {}, result = {}", blobMetadataCache.getCacheId(),
```

Comment:
nit: if it's just constant string you can directly add in the first string input.
reason = Background delete operation
or probably operation = Background delete operation makes more sense.

Reply:
done